src/store/index.js

```diff
@@ -64,6 +64,35 @@ const store = new Vuex.Store({
       });
     },
 
+  //since actions doesn't take more than one argument i add channel to the end of an array (stupid, but it works)
```

Comment:
Actually, this is an object and it's the correct way to parse multiple arguments! 😄 
You can remove this comment if you like, or update it as it's not stupid.